contracts/ownership/HasNoTokens.sol

```diff
@@ -13,11 +13,11 @@ contract HasNoTokens is CanReclaimToken {
 
  /**
   * @dev Reject all ERC23 compatible tokens
-  * @param from_ address The address that is transferring the tokens
-  * @param value_ uint256 the amount of the specified token
-  * @param data_ Bytes The data passed from the caller.
+  * param from_ address The address that is transferring the tokens
```

Comment:
Was there a reason for removing the `@` symbols? Does Solidity not compile with them?